deploy/storage/tables-queues.json

```diff
@@ -1,15 +1,11 @@
 {
   "queues": [
-    "check-complete",
     "check-started",
     "check-submitted",
-    "prepare-check",
     "prepared-check-delete",
```

Comment:
i think we can also delete....
- `prepared-check-delete` queue
- `prepared-check-sync` queue
- `preparedCheck` table

Reply:
Done - I don't think we can remove prepared-check-sync as it seems to be in use.

Reply:
as discussed, you can remove it

Reply:
Done